packages/integrations/tailwind/README.md

```diff
@@ -99,10 +126,31 @@ export default {
   })],
 }
 ```
+</details>
+
+## Examples
+
+- The [Astro Tailwind Starter](https://github.com/withastro/astro/tree/latest/examples/with-tailwindcss?on=github) gets you up and running with a base for your project that uses Tailwind for styling
+- Astro's homepage uses Tailwind. Check out its [Tailwind configuration file](https://github.com/withastro/astro.build/blob/main/tailwind.config.js) or an [example component](https://github.com/withastro/astro.build/blob/main/src/components/integrations/IntegrationCard.astro)
+- The [Astro Ink](https://github.com/one-aalam/astro-ink), [Sarissa Blog](https://github.com/iozcelik/SarissaBlogAstroStarter), and [Creek](https://github.com/robertguss/Astro-Theme-Creek) themes use Tailwind for styling
+- [Browse Astro Tailwind projects on GitHub](https://github.com/search?q=%22%40astrojs%2Ftailwind%22+filename%3Apackage.json&type=Code) for more examples!
 
-## Read more
+## Troubleshooting
+- If your installation doesn't seem to be working, make sure to restart the dev server.
+- If you edit and save a file and don't see your site update accordingly, try refreshing the page.
+- Occasionally, Astro's dev server will run into a problem, and refreshing the page still won't work. In these cases, restart the dev server.
+
+For help, check out the `#support-threads` channel on Discord. Our friendly Support Squad members are here to help!
 
 You can also check our [Astro Integration Documentation][astro-integration] for more on integrations.
 
 [astro-integration]: https://docs.astro.build/en/guides/integrations-guide/
 [astro-ui-frameworks]: https://docs.astro.build/en/core-concepts/framework-components/#using-framework-components
+
+## Contributing
+
+This package is maintained by Astro's Core team. You're welcome to submit an issue or PR!
```

Comment:
Depending on how much we want to encourage this, we could be linking to "new issue", "new PR" here. I know the reader is RIGHT HERE, so not terribly important. But I think a link demonstrates being open to contributions. Just depends if it fits the vibe... and if that's what we in fact want to encourage.

Reply:
This points to a broader problem of the New Issue button being on the _entire_ Astro repo rather than this project... is there a way to link to creating a new issue with "Tailwind Integration:" already filled out in the header?

Reply:
Agreed.  So I checked, and you can enable (and seems like withastro/astro *has* enabled) the You must choose a type of issue screen.

The problem is it looks like this, and only "bug report" actually files an issue in this repo. If you choose documentation, it's a link to the docs repo. 

![Screenshot 2022-05-27 07 44 45](https://user-images.githubusercontent.com/5098874/170685220-1e6fbc06-cfa6-47cd-a04f-a08eaec787ef.png)

The most-proper but I think least-sensible option is to have another choice with bug report that is "correct integration documentation." But that is objectively terrible, especially given that to make issues with REGULAR documentation is already a choice and takes you somewhere different.

So, one imperfect option is to link directly to Docs issues where we will triage, and the workflow becomes to require an issue to Docs to tell us there's something wrong, where we can then guide them through the process if they want to actually submit a PR for it in the proper place. (The default option being that we make the fix ourselves, after the issue alerts us to any errors.)